### PR TITLE
Marathi cheat sheet

### PR DESCRIPTION
**Pull request by mohan08p**
What problem does your Instant Answer solve (Why is it better than organic links)?
It will help to find Basic marathi words and phrases instantly for new people who don't know marathi. It's an Indo-Aryan language spoken predominantly by Marathi people of Maharashtra. It is the official language and co-official language in Maharashtra and Goa states of Western India respectively, and is one of the 23 official languages of India.

What is the data source for your Instant Answer? (Provide a link if possible)
http://www.omniglot.com/language/phrases/marathi.php

Why did you choose this data source?
It's good for learners and also everything is mentioned properly. So, we can go with the track and achieve the milestone while learning.

Are there any other alternative (better) data sources?
not that I know of.

What are some example queries that trigger this Instant Answer?
marathi, marathi phrases, english to marathi, basic marathi phrases, basic marathi

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, voyager, etc)
Communication is better if you are traveling to west india. Every person who want to know a little bit about marathi language.

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?
Yes, https://duck.co/ia/view/marathi_cheat_sheet

Which existing Instant Answers will this one supercede/overlap with?
Noop. This is new IA.

Are you having any problems? Do you need our help with anything?
No.

Developer who created an IA page ?
crash95

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)
marathi

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/language/marathi.json [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+         },
+         {
+            "key" : "Please...",
+            "val" : "कृपया ...",
+            "trn" : "Krupaya..."
+         },
+         {
+            "key" : "Thank you very much!",
+            "val" : "खूप खूप धन्यवाद!",
+            "trn" : "Khup khup dhannywad"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/marathi.json]: @mohan08p lets add ! at end of trn too.
check other blocks also and add necessary !,?

File: share/goodie/cheat_sheets/json/language/marathi.json [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+         {
+            "key" : "Please stop here.",
+            "val" : "येथे थांबवणे.",
+            "trn" : "Yethe thamavane."
+         },
+         {
+            "key" : "Turn right.",
+            "val" : "उजवीकडे वळा.",
+            "trn" : "Ujavikade vala."
+         },
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/marathi.json]: @mohan08p lets remove the periods at end of phrases.
check other blocks too.

Comment (mohan08p) [inline share/goodie/cheat_sheets/json/language/marathi.json]: thanks @jophab

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Marathi Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/marathi_cheat_sheet_mobile.png)

**Comment by jophab** [comment]
@mohan08p This LGTM :+1: 
Are you a marathi speaker?
I dont know marathi so i cant pointout anything in translation.

**Comment by mohan08p** [comment]
@jophab : Yes, I am. It's my native language.

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Comment by mohan08p** [comment]
@moollaza : thanks.

**Pull request merged**